torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1836,28 +1836,32 @@ Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads, const T
   auto gsigma = grads[1];
 
   auto u = raw_u;
-  auto v = raw_v;
+  // TODO: remove .conj()
+  // once https://github.com/pytorch/pytorch/issues/45821 is resolved
```

Comment:
From reading the issue, it is not clear if this is ever going to be fixed.
Maybe mention here that `torch.svd` returns the conjugate of what this function expect, hence the extra conj.

nit: you can move all the conj next to the todo to make it clearer which on  you're talking about in this comment. Or add a detailed comment at one place and reference to it from all the relevant places below.

Reply:
I improved the comment by mentioning that currently SVD returns the conjugate of V.

I think I could resolve the issue soon, the changes should be minimal. It hasn't been fixed yet, because @antocuni  wanted to include it in his `torch.linalg.svd` PR, and then I guess it was decided not to do it there.

Reply:
But changing torch.svd would be a bc-breaking change right? So it would need to go through deprecation, etc?

Reply:
The change concerns only the complex-valued inputs, the support for which was just recently added, and complex support is still in active development so it's not really bc-breaking. This issue was overlooked when complex support was added.

Reply:
Ok that should be fine then. I guess you can move all of these at the top so that they will be easy to remove when we update that. Thanks for the clarification!